mattermost-plugin/webapp/src/index.tsx

```diff
@@ -36,13 +36,13 @@ export default class Plugin {
         this.registry = registry;
         this.channelHeaderButtonId = registry.registerChannelHeaderButtonAction(focalboardIcon, () => {
             const currentChannel = store.getState().entities.channels.currentChannelId;
-            window.open(`${window.location.origin}/plugins/focalboard/workspace/${currentChannel}`)
+            window.open(`${window.location.origin}/plugins/focalboard/workspace/${currentChannel}`);
```

Comment:
We dont use `;`

Reply:
I think I added them because of a linter's complain, but will remove them before the merge